src/components/CheckoutWithCard.tsx

```diff
@@ -75,6 +76,7 @@ export const CheckoutWithCard = ({
         closeModal();
       },
       onOpenKycModal({ iframeLink }) {
+        setShowUnstyledModal(iframeLink.includes('noStyle'));
```

Comment:
if we do remove that unstyled option, we basically can remove this line and the one below